apps/ltn/src/pages/crossings.rs

```diff
@@ -182,11 +180,11 @@ fn help() -> Vec<&'static str> {
     ]
 }
 
-fn boundary_roads(app: &App) -> BTreeSet<RoadID> {
+fn main_roads(app: &App) -> BTreeSet<RoadID> {
```

Comment:
This tool only shows/edits crossings over main roads. Previously it was boundaries of any neighbourhood. But in line with the rest of this PR, main roads are the thing that matters here.